include/lsst/afw/image/PhotoCalib.h

```diff
@@ -74,30 +75,37 @@ inline void assertNonNegative(double value, std::string const &name) {
  * @brief The photometric calibration of an exposure.
  *
  * A PhotoCalib is a BoundedField (a function with a specified domain) that converts from post-ISR
- * counts-on-chip (ADU) to flux and magnitude. It is defined in terms of "maggies", which are a linear
- * unit defined in SDSS (definition given in nanomaggies):
- *     http://www.sdss.org/dr12/algorithms/magnitudes/#nmgy
+ * counts-on-chip (ADU) to flux and magnitude. It is defined such that a calibration of 1 means one count
+ * is equal to one nanojansky (nJy, 10^-35 W/m^2/Hz in SI units). The nJy was chosen because it represents
+ * a linear flux unit with values in a convenient range (e.g. LSST's single image depth of 24.5 is 575 nJy).
+ * See more detailed discussion in: https://pstn-001.lsst.io/
  *
  * PhotoCalib is immutable.
  *
- * The spatially varying flux calibration has units of maggies/ADU, and is defined such that,
+ * The spatially varying flux calibration has units of nJy/ADU, and is defined such that,
  * at a position (x,y) in the domain of the boundedField calibration and for a given measured source instFlux:
  * @f[
- *     instFlux*calibration(x,y) = flux [maggies]
+ *     instFlux*calibration(x,y) = flux [nJy]
  * @f]
  * while the errors (constant on the domain) are defined as:
  * @f[
- *     sqrt((instFluxErr/instFlux)^2 + (calibrationErr/calibration)^2)*flux = fluxErr [maggies]
+ *     sqrt((instFluxErr/instFlux)^2 + (calibrationErr/calibration)^2)*flux = fluxErr [nJy]
```

Comment:
I know this is just updated from before, but blindly adding the calibration error to the instFlux errors (in quadrature) is not often the right thing to do, because of local correlations.  Repeated observations that have been cross-calibrated are not independent samples from the calibrationErr.  Anyway, this is definitely beyond the scope of this ticket.